diff --git a/CMakeLists.txt b/CMakeLists.txt
index 9efaa70..a329b28 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -284,10 +284,6 @@ endif()
 if(FMILIB_BUILD_SHARED_LIB)
 	add_library(fmilib_shared SHARED ${FMILIB_SHARED_SRC})
 
-    if (UNIX AND NOT APPLE)
-        target_compile_definitions(fmilib_shared PRIVATE -D_GNU_SOURCE)
-    endif()
-
 	target_link_libraries(fmilib_shared ${FMILIB_SHARED_SUBLIBS})
 	set(FMILIB_TARGETS ${FMILIB_TARGETS} fmilib_shared)
 endif()
diff --git a/Config.cmake/jmutil.cmake b/Config.cmake/jmutil.cmake
index 1d2a8bd..f329f6f 100644
--- a/Config.cmake/jmutil.cmake
+++ b/Config.cmake/jmutil.cmake
@@ -62,10 +62,6 @@ endif(CYGWIN)
 
 target_link_libraries(jmutils c99snprintf)
 
-if(UNIX AND NOT APPLE)
-    target_compile_definitions(jmutils PRIVATE -D_GNU_SOURCE)
-endif()
-
 if(UNIX)
 	target_link_libraries(jmutils dl)
 endif(UNIX)
